README.md

```diff
@@ -79,7 +79,11 @@ Clone this repository to your desired folder in your laptop:
 
 ### Usage
 
-To open the project, use your prefered explorer and open the index.html file. Or, you can right click the index.html file to open in the live server with the localhost.
+To open the project, you can:
+- Use your prefered explorer and open the index.html file. 
+- You can right click the index.html file to open in the live server with the localhost.
+- Visit this site: https://mpa-mxiang.github.io/profolio2/
```

Comment:
- [ ] Although the link is working as expected, you should use the Live Demo section on the README file for professional documentation, please 🙏🏽 add the LIVE DEMO section on your README and paste the link there and also ensure that all the headers are in the correct position as described in the menu.

![image](https://user-images.githubusercontent.com/58642949/229537072-6a29d9ac-9402-4b35-86fc-1be121d9fcd8.png)